apiserver/common/storagecommon/filesystemdetails.go

```diff
@@ -15,7 +17,9 @@ import (
 
 // FilesystemDetails returns the filesystem and its attachments as a params FilesystemDetails.
 func FilesystemDetails(
+	ctx context.Context,
 	sb DetailsBackend,
+	blockDeviceGetter BlockDeviceGetter,
```

Comment:
nit: BlockDeviceService

Reply:
I used Getter on purpose to reflect the subset of the service api we were actually using here. Similar to what go does with `io.Reader` etc.